e3sm_diags/parser/zonal_mean_2d_parser.py

```diff
@@ -1,43 +1,7 @@
 from e3sm_diags.parameter.zonal_mean_2d_parameter import ZonalMean2dParameter
-
-from .core_parser import CoreParser
+from e3sm_diags.parser.core_parser import CoreParser
 
 
 class ZonalMean2dParser(CoreParser):
     def __init__(self, *args, **kwargs):
-        if "parameter_cls" in kwargs:
-            super().__init__(*args, **kwargs)
-        else:
-            super().__init__(parameter_cls=ZonalMean2dParameter, *args, **kwargs)
-
-    def load_default_args(self, files=[]):
-        # This has '-p' and '--parameter' reserved.
-        super().load_default_args(files)
-
-        # The parameters unique to ZonalMean2dParameter are added here.
-        self.add_argument(
-            "--plevs",
-            type=float,
-            nargs="+",
-            dest="plevs",
-            help="Selected pressure level.[take list as input]",
-            required=False,
-        )
-
-        self.add_argument(
-            "--plot_plevs",
-            dest="plot_plevs",
-            help="plot specified plevs",
-            action="store_const",
-            const=True,
-            required=False,
-        )
-
-        self.add_argument(
-            "--plot_log_plevs",
-            dest="plot_log_plevs",
-            help="plot plevs on log-scale",
-            action="store_const",
-            const=True,
-            required=False,
-        )
```

Comment:
I removed these arguments from `ZonalMean2dParser` because they already exist in `CoreParser`.

Do we still need to define a separate `ZonalMean2dParser` and `ZonalMean2dStratosphereParser` if they are identical to `CoreParser`?

Reply:
Hey @tomvothecoder , the original design was that some diagnostics set has their own set-specific parameters/parsers that are not needed for the core set.  And any parameters specified under those special sets would overwrite set parameters in the parent object. This design makes is possible to co-operate all the sets: for instance, those coresets requires climatology files as input, while sets like QBO, ENSO requires monthly time-series files. In this case we need the capability to overwrite parameters, such as test/obs data path. Originally parameter and parser objects are written consistently (mostly).

Reply:
Yes, we needs the capability for `ZonalMean2dParser` to overwrite what is already defined in `CoreParser`. In this specific example, `--plevs` should be in both, but `--plot_plevs`, and `--plot_log_plevs` are only applicable for `ZonalMean2dParser`.

Reply:
Gotcha, I'll restore these args for `ZonalMean2dParser`